data/vapor_pressure/vapor_pressure.py

```diff
@@ -0,0 +1,12 @@
+import math
+h2o_smiles = "O"
+a = 0.61094
+b = 17.635
+c = 243.04
+T = 25
+
+exp_1 = ((b*T)/(T+c))
+exp_2 = math.exp(exp_1)
+vp_calc = a*exp_2
+
+result = True if vapor_pressure("O")==vp_calc else False
```

Comment:
Looks great! One last thing, use isclose() to compare floating point numbers because they can never be exactly the same due to rounding/truncation errors.

Reply:
Fixed!